Solaris libXp was shipped as libXp.so.1 so use that for binary compatibility

--- ./src/Makefile.am	Fri Dec  2 20:41:50 2005
+++ ./src/Makefile.am	Mon Aug 11 14:49:19 2008
@@ -31,5 +31,4 @@
 # every platform, so until we change the ABI in an incompatible fashion,
 # we have to use -version-number
 #
-XP_VERSION=-version-number 6:2:0
-libXp_la_LDFLAGS = $(XP_VERSION) -no-undefined
+libXp_la_LDFLAGS = -version-number 1 -no-undefined
